website/content/en/status/report-2023-01-2023-03/cloud-init.adoc

```diff
@@ -16,6 +16,10 @@ This quarter has been going very, very slowly, for personal reasons — but also
 I have been trying to link:https://github.com/canonical/cloud-init/pull/2003[port the Infiniband functions].
 This has proven difficult, because it falsified my thesis that man:ifconfig[8] is all that is needed to figure out network interfaces on FreeBSD.
 
+While waiting for resources, I link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=269767[debugged a boot panic] and got it fixed: gitref:499171a98c8813e4dc6e085461d5c47750efa555[repository=src].
+This now makes it possible to boot FreeBSD on LXD - cloud-init's CI platform.
```

Comment:
A pair of hyphens: 

> … LXD -- cloud-init's …

– will be rendered as an em dash. 

<https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#text-replacements>